tensorboard/webapp/widgets/line_chart_v2/lib/public_types.ts

```diff
@@ -0,0 +1,26 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+export {
+  DataSeries,
+  DataSeriesMetadata,
+  DataSeriesMetadataMap,
+  Dimension,
+  Extent,
+  Point,
+  Rect,
+} from './internal_types';
```

Comment:
Looking ahead at [1], I see evidence why most of these need to be public, except for `Rect`.  It seems that `Rect` is useful for anything that deals with a `Coordinator`, which line_chart_component and above should not be exposed to.

Should we drop `Rect`?

[1] https://github.com/tensorflow/tensorboard/blob/swl_gpu_11/tensorboard/webapp/widgets/line_chart_v2/line_chart_component.ts

Reply:
Done